include/deal.II/cgal/surface_mesh.h

```diff
@@ -66,6 +94,50 @@ namespace CGALWrappers
     const typename dealii::Triangulation<dim, spacedim>::cell_iterator &cell,
     const dealii::Mapping<dim, spacedim> &                              mapping,
     CGAL::Surface_mesh<CGALPointType> &                                 mesh);
+
+  /**
+   * Performs corefinement and boolean operations on two deal.II cells and put
+   * the result in a CGAL::Surface_mesh object. If the cells are disjoint and
+   * you try to perform a boolean operation which makes no sense, like their
+   * intersection, an exception is thrown.
```

Comment:
Do we want to throw an exception, or simply return an empty mesh? In the end, an empty set is a valid result for an intersection from a mathematical standpoint.